icaruscode/CRT/crthitproducer.fcl

```diff
@@ -16,6 +16,198 @@ standard_crthitrecoalg:
+		      [74,2000283],
+		      [75,2000303],
+		      [76,2000323],
+		      [79,2000343],
+		      [78,2000363],
+		      [77,2000383],
+		      [81,2000263],
+		      [80,2000283],
+		      [83,2000329],
+		      [84,2000349],
+		      [85,2000369],
+		      [82,2000415]
+    ]
+    FEB_T0delay_side: [
+       [53,240],
+                  [51,260],	
+                  [49,280],
+                  [50,330],
+                  [52,350],
+                  [54,370],
+                  [59,240],
+                  [57,260],
+                  [55,280],
+                  [56,330],
+                  [58,350],
+                  [60,370],
+                  [65,230],
+                  [63,250],
+                  [61,270],
+                  [62,316],
+                  [64,336],
+                  [66,356],
+                  [71,230],
+                  [69,250],
+                  [67,270],
+                  [68,316],
+                  [70,336],
+                  [72,356],
+                  [41,245],
+                  [39,265],
+                  [37,285],
+                  [38,331],
+                  [40,351],
+                  [42,371],
+                  [47,245],
+                  [45,265],
+                  [43,285],
+                  [44,331],
+                  [46,351],
+                  [48,371],
+                  [88,364],
+                  [92,384],
+                  [93,404],
+                  [89,450],
+                  [91,433],
+                  [87,453],
+                  [86,470],
+                  [90,490],
+                  [81,237],
+                  [80,257],
+                  [83,303],
+                  [84,323],
+                  [85,343],
+                  [82,390],
+                  [73,237],
+                  [74,257],
+                  [75,277],
+                  [76,297],
+                  [79,317],
+                  [78,337],
+                  [77,357],
+                  [17,359],
+                  [15,379],
+                  [13,399],
+                  [14,448],
+                  [16,468],
+                  [18,488],
+                  [23,359],
+                  [21,379],
+                  [19,399],
+                  [20,448],
+                  [22,468],
+                  [24,488],
+                  [29,348],
+                  [27,368],
+                  [25,388],
+                  [26,434],
+                  [28,454],
+                  [30,474],
+                  [35,348],
+                  [33,368],
+                  [31,388],
+                  [32,434],
+                  [34,454],
+                  [36,474],
+		  [6,364],
+                  [4,384],
+                  [2,404],
+                  [1,450],
+                  [3,470],
+                  [5,490],
+                  [12,364],
+                  [10,384],
+                  [8,404],
+                  [7,450],
+                  [9,470],
+                  [11,490]
+    ]  
```

Comment:
Thank you Anna for m,oving these delays to a fcl Parameters. This has to be done for Top CRT, I will wait for this PR to be included before doing this. Can I just ask you if it is possible to move T1 and T0 delays to a common 3 values struct and not 2 different struct? The index is the same (FEB) board number, putting T0 and T1 delays should provide, also, a visual check if the values make sense by comparison of the two